src/components/publishing/display/canvas/canvas_video.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import styled from "styled-components"
+import { pMedia } from "../../../helpers"
+
+interface VideoProps {
+  src: any
+}
+
+class DisplayCanvasVideo extends React.Component<VideoProps, any> {
+  constructor(props) {
+    super(props)
+    this.state = { isPlaying: false }
+  }
+
+  onPlayVideo = e => {
+    e.preventDefault()
+    e.target.paused ? e.target.play() : e.target.pause()
```

Comment:
I would be really careful using ternaries in this way, especially without semicolons -- they're typically for variable assignment and can lead to strange side-effects if used otherwise. 

```js
if (e.target.paused) {
  e.target.play()
} else {
  e.target.pause()
}
```
In most eslint settings theres a rule that prevent that kind of use and enforce the above; we should probably turn it on in tslint.